tools/ktx/command_create.cpp

```diff
@@ -820,7 +822,7 @@ Create a KTX2 file from various input files.
 */
 class CommandCreate : public Command {
 private:
-    Combine<OptionsCreate, OptionsASTC, OptionsCodec<false>, OptionsMetrics, OptionsCompress, OptionsMultiInSingleOut, OptionsGeneric> options;
+    Combine<OptionsCreate, OptionsASTC, OptionsCodec<false>, OptionsCodecCommon, OptionsMetrics, OptionsCompress, OptionsMultiInSingleOut, OptionsGeneric> options;
```

Comment:
OptionsBasis or OptionsBasisCodec.

Reply:
Not sure what you mean. I don't see OptionsBasis. Do you mean to rename this OptionsCode<false> to OptionsBasis?

Reply:
Fixed in https://github.com/KhronosGroup/KTX-Software/pull/817/commits/5a43fdfebba8553168c5e7df13cf58c2d51332fd